python/cuml/test/experimental/test_explainer_kernel_shap.py

```diff
@@ -51,6 +51,29 @@ def exact_tests_dataset():
     return X_train, X_test, y_train, y_test
 
 
+def experimental_test_and_log(cu_shap_values,
+                              golden_result_values,
+                              fx,
+                              expected,
+                              tolerance=1e-02):
+    # while experimental we test the golden results but also fall back to
+    # testing the sum of the values to avoid instability in CI
+    close_values = \
+        np.allclose(cu_shap_values, golden_result_values,
+                    rtol=tolerance, atol=tolerance)
+
+    expected_sum = 0.99 <= (abs(np.sum(cp.asnumpy(
+        cu_shap_values))) / abs(fx - expected)) <= 1.01
+
+    if not close_values:
```

Comment:
do we want the abs? seems like there should not be a sign flip?

how about just `np.allclose(1.00, np.sum(cp.asnumpy(..)) / (fx - expected), atol=tolerance)` so we're consistent with other tolerance use?